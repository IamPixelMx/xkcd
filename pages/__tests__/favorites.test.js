import React from 'react';
import configureStore from 'redux-mock-store';

import { renderWithProviderSnapshot } from 'jest-utils';
import Favorites from '../favorites';

const mockStore = configureStore([]);

describe('Favorites', () => {
  const favoritesElement = renderWithProviderSnapshot(<Favorites />, { mockStore }).toJSON();

  it('Renders Favorites page', () => {
    expect(favoritesElement).toMatchSnapshot();
  });
});
