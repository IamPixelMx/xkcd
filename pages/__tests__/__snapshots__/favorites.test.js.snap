// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites Renders Favorites page 1`] = `
<main
  className="container"
>
  <h1
    className="title"
  >
    Favoritos
  </h1>
</main>
`;
